Models/HealthCareProvider.js

```diff
@@ -0,0 +1,76 @@
+module.exports = (sequelize, DataTypes) => {
+  const HealthCareProvider = sequelize.define(
+    "HealthCareProvider",
+    {
+      id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+      },
+      name: {
+        type: DataTypes.STRING(255),
+      },
+      address_line1: {
+        type: DataTypes.TEXT,
+      },
+      accreditation_no: {
+        type: DataTypes.STRING,
+      },
+      hotline: {
+        type: DataTypes.STRING,
+      },
+      private_govt: {
+        type: DataTypes.ENUM,
+        values: ["private", "government"],
+      },
+      occupation: {
```

Comment:
There should be no occupation field for Health care providers

Reply:
Which of the fields? or do you mean everything?